src/main/java/com/lorepo/icplayer/client/module/text/InlineChoiceWidget.java

```diff
@@ -233,6 +233,17 @@ public String getId() {
 		return choiceInfo.getId();
 	}
 
+    @Override
+	public boolean getResetStatus() {
+// 		return choiceInfo.getResetStatus();
+        return false;
+	}
+
+	@Override
+	public void setResetStatus(boolean wasReset) {
+// 		choiceInfo.setResetStatus(wasReset);
```

Comment:
W nawiązaniu do naszej rozmowy należy zmienić nazwę komentarza na `TODO not implement`